app/javascript/shared/components/useKeyboardShortcuts.js

```diff
@@ -12,18 +12,22 @@ function isFormField(element) {
   if (element instanceof HTMLElement === false) return false;
 
   const name = element.nodeName.toLowerCase();
-  const type = (element.getAttribute("type") || "").toLowerCase();
+  const type = (element.getAttribute('type') || '').toLowerCase();
   return (
-    name === "select" ||
-    name === "textarea" ||
-    (name === "input" && type !== 'submit' && type !== 'reset' && type !== 'checkbox' && type !== 'radio') ||
+    name === 'select' ||
+    name === 'textarea' ||
+    (name === 'input' &&
+      type !== 'submit' &&
+      type !== 'reset' &&
+      type !== 'checkbox' &&
+      type !== 'radio') ||
     element.isContentEditable
   );
 }
 
 // Default options to be used if null
 const defaultOptions = {
-  timeout: 500
+  timeout: 0,
```

Comment:
This is the actual fix. The rest is prettier in action.

Reply:
Should we improve the documentation here to include a reason why we are explicitly having the default timeout be `0` here? Just thinking that we might want to document that for our future selves since it is intentional. But not a blocker; let me know what you think